docs/pages/try-out-teleport/linux-server.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Deploy Open Source Teleport on a Linux Server
+title: Deploy Teleport on a Linux Server
```

Comment:
```suggestion
title: Try out Teleport on a Linux Server
```
One thing I'd suggest in order to clarify that this guide is for trying out Teleport, not for production use cases. [One user](https://github.com/gravitational/teleport/issues/19107) got confused as to why were were instructing them to create an HTML page when attempting to deploy to production, for example.